main/src/main/java/com/google/android/apps/muzei/util/SvgPathParser.java

```diff
@@ -149,6 +149,21 @@ public Path parsePath(String s) throws ParseException {
                     mCurrentPoint.set(tempPoint1);
                     break;
                 }
+                
```

Comment:
I explicitly left the extra space on purpose because that is the format of the rest of the file, isn't it? Look at the extra spaces between every case clause. Am I missing something?